src/fetchRepo.js

```diff
@@ -1,49 +1,39 @@
-const axios = require("axios");
+const { request } = require("./utils");
 
 async function fetchRepo(username, reponame) {
   if (!username || !reponame) {
     throw new Error("Invalid username or reponame");
   }
 
-  const res = await axios({
-    url: "https://api.github.com/graphql",
-    method: "post",
-    headers: {
-      Authorization: `bearer ${process.env.GITHUB_TOKEN}`,
-    },
-    data: {
-      query: `
-        fragment RepoInfo on Repository {
-          name
-          stargazers {
-            totalCount
-          }
-          description
-          primaryLanguage {
-            color
-            id
-            name
-          }
-          forkCount
+  const res = await request(`
+    fragment RepoInfo on Repository {
+      name
+      stargazers {
+        totalCount
+      }
```

Comment:
Can we change the API to be like this?

```js
const res = await request({
  query: `...`,
  variables: {...}
})
```

because it's more explicit and i like explicitness

Reply:
Yes, sure!